src/components/sendV2/send.test.js

```diff
@@ -104,15 +116,39 @@ describe('Form', () => {
     expect(wrapper).toContainMatchingElement('.send-box');
     expect(wrapper).toContainMatchingElement('MultiStep');
     expect(wrapper).toContainMatchingElement('Form');
+    expect(wrapper).not.toContainMatchingElement('Summary');
+    expect(wrapper).not.toContainMatchingElement('TransactionStatus');
   });
 
   it('should render properly without getting data from URL', () => {
-    props.history.location.path = '';
-    props.history.location.search = '';
-    wrapper = mount(<Send {...props} />, options);
+    const newProps = { ...props };
+    newProps.history.location.path = '';
+    newProps.history.location.search = '';
+    wrapper = mount(<Send {...newProps} />, options);
     wrapper.update();
     expect(wrapper).toContainMatchingElement('.send-box');
     expect(wrapper).toContainMatchingElement('MultiStep');
     expect(wrapper).toContainMatchingElement('Form');
   });
+
+  it('should call finallCallback after submit a transaction', () => {
+    const bkm = { target: { name: 'recipient', value: '12345L' } };
+    const amount = { target: { name: 'amount', value: '.1' } };
+
+    wrapper.find('input.recipient').simulate('change', bkm);
+    const amountField = wrapper.find('.fieldGroup').at(1);
+    amountField.find('InputV2').simulate('change', amount);
```

Comment:
Since you just use this variable here, is it really needed?

Reply:
done